calcom/cal.com
# fix: remove app dialog while removing installed calendar

## What does this PR do?

This PR fixes the alignment of remove app dialog while removing an installed calendar.

Fixes #10179 


https://github.com/calcom/cal.com/assets/20976813/70ea1588-bcb7-4d76-9e2f-ccd43648ae69


## Requirement/Documentation

N/A

## Type of change


## How should this be tested?

  Try to remove an installed calendar app. The alignment of the remove app dialog should be as per designs. 

## Mandatory Tasks


## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
